src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -417,45 +419,45 @@ item.appliedenergistics2.material.Singularity.name=Singularität
 item.appliedenergistics2.material.Wireless.name=Drahtlosempfänger
 item.appliedenergistics2.material.WirelessBooster.name=Drahtlosverstärker
 item.appliedenergistics2.material.WoodenGear.name=Holzzahnrad
-item.appliedenergistics2.material.EngProcessorPress.name=Gravur-Konstruktionsdruck
-item.appliedenergistics2.material.CalcProcessorPress.name=Gravur-Kalkulationsdruck
-item.appliedenergistics2.material.LogicProcessorPress.name=Gravur-Logikdruck
-item.appliedenergistics2.material.EngProcessorPrint.name=Gedruckter Konstruktionsschaltkreis
-item.appliedenergistics2.material.CalcProcessorPrint.name=Gedruckter Kalkulationsschaltkreis
-item.appliedenergistics2.material.LogicProcessorPrint.name=Gedruckter Logikschaltkreis
-item.appliedenergistics2.material.SiliconPress.name=Gravur-Siliziumdruck
-item.appliedenergistics2.material.SiliconPrint.name=Gedrucktes Silizium
-item.appliedenergistics2.material.NamePress.name=Schreiber-Namensdruck
+item.appliedenergistics2.material.EngProcessorPress.name=Konstruktionsschaltkreisschablone
+item.appliedenergistics2.material.CalcProcessorPress.name=Kalkulationsschaltkreisschablone
+item.appliedenergistics2.material.LogicProcessorPress.name=Logikschaltkreisschablone
+item.appliedenergistics2.material.EngProcessorPrint.name=Gepresster Konstruktionsschaltkreis
+item.appliedenergistics2.material.CalcProcessorPrint.name=Gepresster Kalkulationsschaltkreis
+item.appliedenergistics2.material.LogicProcessorPrint.name=Gepresster Logikschaltkreis
+item.appliedenergistics2.material.SiliconPress.name=Siliziumschablone
+item.appliedenergistics2.material.SiliconPrint.name=Gepresstes Silizium
+item.appliedenergistics2.material.NamePress.name=Namensschablone
 item.appliedenergistics2.material.SkyDust.name=Himmelssteinstaub
 item.appliedenergistics2.material.CardCrafting.name=Fertigungskarte
 
-item.appliedenergistics2.multi_part.AnnihilationPlane.name=ME-Annihilationsfeld
-item.appliedenergistics2.multi_part.IdentityAnnihilationPlane.name=ME-Identitätsannihilationsfelds
+item.appliedenergistics2.multi_part.AnnihilationPlane.name=ME Annihilationsfeld
+item.appliedenergistics2.multi_part.IdentityAnnihilationPlane.name=ME Identitätsannihilationsfelds
 item.appliedenergistics2.multi_part.CableAnchor.name=Kabelanker
-item.appliedenergistics2.multi_part.CableCovered.name=Verdecktes ME-Kabel
+item.appliedenergistics2.multi_part.CableCovered.name=Verdecktes ME Kabel
 item.appliedenergistics2.multi_part.CableGlass.name=ME-Glaskabel
-item.appliedenergistics2.multi_part.CableSmart.name=Schlaues ME-Kabel
-item.appliedenergistics2.multi_part.CableDense.name=Dichtes ME-Kabel
-item.appliedenergistics2.multi_part.StorageMonitor.name=ME-Speichermonitor
-item.appliedenergistics2.multi_part.ConversionMonitor.name=ME-Konversionsmonitor
-item.appliedenergistics2.multi_part.CraftingTerminal.name=ME-Fertigungskonsole
+item.appliedenergistics2.multi_part.CableSmart.name=Schlaues ME Kabel
+item.appliedenergistics2.multi_part.CableDense.name=Dickes ME Kabel
```

Comment:
Same with Energy Cell, used "Dick" here cause it made more sense here
I´ll change it back to "Dicht" until someone comes up with a better translation